BuildScript/BuildTasks/ExecutionRuntime.cs

```diff
@@ -23,12 +23,22 @@ public class ExecutionRuntime
     public bool UseDocker { get; }
     public string Key { get; }
     public string Value { get; }
+    public string DockerImage { get; }
 
-    public ExecutionRuntime (string key, string value, bool useDocker)
+    public ExecutionRuntime (string key, string value, bool useDocker, string dockerImage)
     {
       Key = key;
       Value = value;
       UseDocker = useDocker;
+      DockerImage = dockerImage;
+    }
+
+    public ExecutionRuntime (string sameKeyAndValue, bool useDocker, string dockerImage)
```

Comment:
I think we can drop that overload. it's not really all that helpful on the callsite. also, when doing somehting like this, delgate to the same ctor.